File: lib/js/js_exn.js
'use strict';

Caml_js_exceptions = require("./caml_js_exceptions.js");

function raiseError(str) do
  throw new Error(str);
end end

function raiseEvalError(str) do
  throw new EvalError(str);
end end

function raiseRangeError(str) do
  throw new RangeError(str);
end end

function raiseReferenceError(str) do
  throw new ReferenceError(str);
end end

function raiseSyntaxError(str) do
  throw new SyntaxError(str);
end end

function raiseTypeError(str) do
  throw new TypeError(str);
end end

function raiseUriError(str) do
  throw new URIError(str);
end end

$$Error$1 = Caml_js_exceptions.$$Error;

exports.$$Error = $$Error$1;
exports.raiseError = raiseError;
exports.raiseEvalError = raiseEvalError;
exports.raiseRangeError = raiseRangeError;
exports.raiseReferenceError = raiseReferenceError;
exports.raiseSyntaxError = raiseSyntaxError;
exports.raiseTypeError = raiseTypeError;
exports.raiseUriError = raiseUriError;
--[ No side effect ]--

File: lib/es6/js_exn.lua


local Caml_js_exceptions = require "..caml_js_exceptions.lua";

function raiseError(str) do
  error(new __Error(str))
end end

function raiseEvalError(str) do
  error(new __EvalError(str))
end end

function raiseRangeError(str) do
  error(new __RangeError(str))
end end

function raiseReferenceError(str) do
  error(new __ReferenceError(str))
end end

function raiseSyntaxError(str) do
  error(new __SyntaxError(str))
end end

function raiseTypeError(str) do
  error(new __TypeError(str))
end end

function raiseUriError(str) do
  error(new __URIError(str))
end end

__Error_1 = Caml_js_exceptions.__Error;

export do
  __Error_1 as __Error,
  raiseError ,
  raiseEvalError ,
  raiseRangeError ,
  raiseReferenceError ,
  raiseSyntaxError ,
  raiseTypeError ,
  raiseUriError ,
  
end
--[[ No side effect ]]
